core/src/mindustry/mod/Mods.java

```diff
@@ -749,7 +749,7 @@ public int getMinMinor(){
                 }
             }
 
-            return Strings.parseInt(ver, 0);
+            return 0;
```

Comment:
@Anuken i revert this because this "fix" do nothing
https://github.com/Anuken/Mindustry/commit/3e2fcfa30fc721713e3c6566ee752a00db427741#diff-fe3c06f2dfc0d0e74932cfb43380e993R752

Reply:
see https://github.com/Anuken/Mindustry/pull/2531/commits/26feeb96d3a03b7ff212edcf26577e66346c2124 and https://github.com/Anuken/Mindustry/pull/2531/commits/6875e8d055e284e2709cbab8bd6c1e438e4cfe53

Reply:
No, it *does* do something. It's just not obvious because you didn't set a min version.

Reply:
> 
> 
> No, it _does_ do something. It's just not obvious because you didn't set a min version.

Yep. The _`getMinMajor()`_ function will return the **major** version if you don't specify a **minor** version after dot in field _`minGameVersion`_ in **mod.json**